src/main/java/org/terasology/logic/health/RestorationAuthoritySystem.java

```diff
@@ -28,6 +28,17 @@
 import org.terasology.logic.players.event.OnPlayerRespawnedEvent;
 import org.terasology.math.TeraMath;
 
+/**
+ * This system takes care of restoration of entities with HealthComponent.
+ * To increase the health of an entity, send DoRestoreEvent
+ * <p>
+ * Logic flow for restoration:
+ * - DoRestoreEvent
```

Comment:
This is too big to consider as part of this PR (since the same word is in use many other places) but I wonder if `do` is a good term in general for the starting phase of an event chain like this one. "Do" to me very much means that the thing is happening right now. Maybe "Perform" would give a better impression that it is more like an action that is being performed, almost like a stage play might be going through several phases to arrive at an action :-)

This could be overly subjective as interpreting these kinds of words may differ subtly between individuals.